components/expenses/Expense.js

```diff
@@ -109,6 +121,14 @@ class Expense extends React.Component {
       },
       no: { id: 'no', defaultMessage: 'No' },
       yes: { id: 'yes', defaultMessage: 'Yes' },
+      'delete.modal.header': {
+        id: 'delete.modal.header',
+        defaultMessage: 'Delete Expense',
+      },
+      'delete.modal.body': {
+        id: 'delete.modal.body',
```

Comment:
Id should not be `delete.modal.body` if this is about deleting an expense.

```suggestion
        id: 'deleteExpense.modal.body',
```